static/js/components/EditFollowupRequestDialog.jsx

```diff
@@ -40,6 +40,27 @@ const EditFollowupRequestDialog = ({
     handleClose();
   };
 
+  // Set default form values to current request data
+  const { formSchema } = instrumentFormParams[
+    followupRequest.allocation.instrument.id
+  ];
+
+  Object.keys(formSchema.properties).forEach((key) => {
+    formSchema.properties[key].default = followupRequest.payload[key];
```

Comment:
Why is the default being set to the payload value?  A code comment would be useful here.

Reply:
Done